### DISCUSSION TITLE
support multiple datasources

### DISCUSSION THREAD

**Discussion by ivan1410** [mayooear/ai-pdf-chatbot-langchain#36] [Ideas]
Every day, we generate a vast amount of data across various platforms, including emails, PDFs, Word documents, and other cloud-based tools. I was contemplating the idea of having this tool support multiple data sources. Perhaps even enabling the tool to connect with APIs from other data sources. Is it feasible to incorporate such functionality into this project?

**Comment by mayooear** [comment]
Yes it is, but I'd like to handle one feature at a time. What is the most important data source for your usecase?
